it/v3/endpoints/TriggerBsasControllerISpec.scala

```diff
@@ -19,47 +19,56 @@ package v3.endpoints
 import com.github.tomakehurst.wiremock.stubbing.StubMapping
 import play.api.http.HeaderNames.ACCEPT
 import play.api.http.Status._
-import play.api.libs.json.{JsObject, Json}
-import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.libs.json.{ JsObject, Json }
+import play.api.libs.ws.{ WSRequest, WSResponse }
 import play.api.test.Helpers.AUTHORIZATION
 import support.IntegrationBaseSpec
 import v3.fixtures.TriggerBsasRequestBodyFixtures._
 import v3.models.errors._
-import v3.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+import v3.stubs.{ AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub }
 
 class TriggerBsasControllerISpec extends IntegrationBaseSpec {
 
   private trait Test {
 
     val nino = "AA123456A"
 
-    def setupStubs(): StubMapping
+    def taxYear: String
     def uri: String = s"/$nino/trigger"
-    def desUrl: String = s"/income-tax/adjustable-summary-calculation/$nino"
+    def downstreamUri: String
+    def downstreamTaxYear: String
```

Comment:
can we move this under `taxYear` so the taxYears are grouped and the uris are grouped